docker-compose.yml

```diff
@@ -195,19 +198,47 @@ services:
     volumes:
       - pgdata:/var/lib/postgresql/data
 
+  #---------------------------------------------------------------------------#
+  # Official Grafana Image                                                     #
+  #---------------------------------------------------------------------------#
+  # # Comment out to use
+  #
+  # grafana:
+  #   image: grafana/grafana:7.1.0-beta1
+  #   depends_on:
+  #     - timescaledb
+  #     - radar-jdbc-connector
+  #   user: '1000'
+  #   environment:
+  #     GF_SECURITY_ADMIN_PASSWORD: ${GRAFANA_PASSWORD}
+  #     GF_SERVER_ROOT_URL: ${GRAFANA_URL}
+  #     GF_USERS_VIEWERS_CAN_EDIT: 'true'
+  #     GF_INSTALL_PLUGINS: 'flant-statusmap-panel'
+  #     GF_USERS_DEFAULT_THEME: light
+  #     GF_DASHBOARDS_DEFAULT_HOME_DASHBOARD_PATH: '/etc/grafana/provisioning/dashboards/home.json'
+  #     POSTGRES_DB: ${POSTGRES_DB}
+  #     POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+  #     POSTGRES_HOST: timescaledb:5432
+  #   ports:
+  #     - '3001:3000'
+  #   volumes:
+  #     - grafana-lib:/var/lib/grafana
+  #     - grafana-log:/var/log/grafana
+  #     - ./grafana/:/etc/grafana/provisioning/
+
+  #---------------------------------------------------------------------------#
+  # RADAR Docker Image                                                       #
+  #---------------------------------------------------------------------------#
   grafana:
-    image: grafana/grafana:7.1.0-beta1
+    image: radar-grafana
+    build: .
```

Comment:
It's good practice to always add a `.dockerignore` file here. Include at least `.git`, and possibly also other files that won't be needed during the build (e.g. `docker`).

Also, since the bash files in the `docker` directory aren't used at all, those could be removed entirely.